### ISSUE TITLE
Accessibility menu inaccessible from gamepad/keyboard

### ISSUE THREAD

**Issue by tanuki-billie** [nzp-team/nzportable#1082]
### System Model

Linux (x86_64)

### Game Version

2.0.0-indev+20250122072046

### Download Source

https://nzp-team.itch.io/nazi-zombies-portable

### Description

When attempting to select accessibility options by using a gamepad or a keyboard, the option is skipped (you skip from Controls to Open Console).

### Steps to reproduce

1. Open the Configuration menu
2. Attempt to arrow down to Accessibility
3. Weep as one cannot access the accessibility menu?

### Verification of Submission

- [x] I agree that I properly filled out the items listed above. I understand an issue may be closed if process is not properly followed. I will attach an image or video below this issue after pressing submit.

**Comment by tanuki-billie** [comment]
https://github.com/user-attachments/assets/6f753b53-9c8a-450f-a0ef-ecb1bcb21b9e
